packages/node_modules/nav-frontend-popover/_popover-normal.example.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Knapp } from 'NavFrontendModules/nav-frontend-knapper';
+import { guid } from 'NavFrontendModules/nav-frontend-js-utils';
+// eslint-disable-next-line
+import Popover from './';
+
+export default class PopoverNormalEksempel extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            anker: undefined
+        };
+
+        this.popoverId = guid();
+    }
+
+    togglePopover = (anker) => {
+        const newState = Object.assign({}, this.state);
+        newState.anker = (newState.anker) ? undefined : anker;
+        this.setState(newState);
+    }
+
+    render() {
+        return (
+            <div>
+                <Knapp
+                    onClick={(e) => this.togglePopover(e.currentTarget)}
+                    aria-expanded={this.state.anker !== undefined}
+                    aria-controls={this.popoverId}
+                    aria-haspopup="dialog"
+                >
+                    Åpne popover
+                </Knapp>
+                <Popover
+                    id={this.popoverId}
+                    ankerEl={this.state.anker}
```

Comment:
Er det ikke litt merksnolig at `this.state.anker` byttes mellom knappen og undefined basert på om den er åpen eller ikke?

Reply:
Hm, tenker kanskje det vil være meningsløst å kunne angi `åpen={true}` uten å samtidig angi et anker - og at ankeret derfor like gjerne kan fungere som `åpen`-prop samtidig. Men godt mulig det bare gjør APIet forvirrende og at besparelsen av en ekstra prop ikke er verd det 🤔

Reply:
Ah, så var det jo fordelen av å kunne sende inn anker-elementet med `event.currentTarget` på denne måten. Hvis vi skriver om dette til å være `ref`-basert; vil ikke det medføre at vi må utbedre alle potensielle anker-komponenter med forwarded refs først? 😕